### PR TITLE
Do not set SELinux to permissive when it is enabled in check mode

### PR DESCRIPTION
**Pull request by dlouks**
…ning in check mode

### Proposed changes

This change is to suppress reported changes when running in check mode and selinux is enabled.
[Open Issue](https://github.com/nginxinc/ansible-role-nginx-config/issues/211)

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/ansible-role-nginx-config/blob/main/CONTRIBUTING.md) document
- [x] I have added Molecule tests that prove my fix is effective or that my feature works
- [x] I have checked that any relevant Molecule tests pass after adding my changes
- [x] I have updated any relevant documentation (`defaults/main/*.yml`, `README.md` and `CHANGELOG.md`)

### CODE REVIEW

**Comment by alessfg** [comment]
There's a bug with Ansible lint at the moment that is causing all tests to fail. I'll rerun these tests once I fix it.

**Comment by alessfg** [comment]
Looks like tests are passing! Can you add a note to the Changelog?

File: tasks/prerequisites/setup-selinux.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@
   selinux:
     state: permissive
     policy: targeted
+  when: not (ansible_check_mode and nginx_selinux_enforcing)
```
Comment (alessfg) [inline tasks/prerequisites/setup-selinux.yml:24]: This should be `nginx_config_selinux_enforcing`

**Review by alessfg — CHANGES_REQUESTED** [review]
I'd be great to also update the Changelog while at it

File: tasks/prerequisites/setup-selinux.yml (hunk 1 above)
Comment (dlouks) [reply tasks/prerequisites/setup-selinux.yml]: Thanks for catching! Updates made.

**Review by magicalyak — APPROVED** [review]

**Review by alessfg — APPROVED** [review]

**Pull request merged**